lib/routes/project.js

```diff
@@ -105,6 +105,30 @@ module.exports = function (modulus) {
       modulus.runCommand(modulus.commands.project.delete, [modulus.program.projectName], true);
     });
 
+  // resize command
+  help.add('resize', function () {
+    this.line('project resize'.verbose);
+    this.line('Changes a project\'s servo size.'.input);
+    this.line('Please note that the project must be stopped and started for the changes to take effect.'.input)
+    this.line('  options:'.input);
+    this.line('    -p, --project-name    Name of the project to resize. Prompts are skipped when specified.'.input);
+    this.line('    -s, --servo-size      New servo size [192, 396, 512, 1024, 2048].'.input);
+  });
+
+  modulus.program
+    .command('project resize')
+    .description('Changes a project\'s servo size.')
```

Comment:
In the description add some type of note that will inform the user their project needs to be stop/started after successful update.